lib/simulator-xcode-8.js

```
@@ -282,30 +283,9 @@ class SimulatorXcode8 extends SimulatorXcode7 {
    * @throws {Error} If there was an error while setting the location
    */
   async setGeolocation (latitude, longitude) {
-    if (this.idb) {
-      await this.idb.setLocation(latitude, longitude);
-      return true;
-    }
-    log.warn(`Cannot set geolocation with idb, because it is not installed. Defaulting to AppleScript`);
-    const output = await this.executeUIClientScript(`
-      tell application "System Events"
-        tell process "Simulator"
-          set featureName to "Custom Location"
-          set dstMenuItem to menu item (featureName & "…") of menu 1 of menu item "Location" of menu 1 of menu bar item "Debug" of menu bar 1
-          click dstMenuItem
-          delay 1
-          set value of text field 1 of window featureName to ${latitude} as string
-          delay 0.5
-          set value of text field 2 of window featureName to ${longitude} as string
-          delay 0.5
-          click button "OK" of window featureName
-          delay 0.5
-          set isInvisible to (not (exists (window featureName)))
-        end tell
-      end tell
-    `);
-    log.debug(`Geolocation parameters dialog accepted: ${output}`);
-    return _.isString(output) && output.trim() === 'true';
+    return await execLyftSetLocation(this.udid, latitude, longitude)
```

The method should throw an error if all the possible location methods failed